vnext/src/index.windows.js

```diff
@@ -102,6 +102,7 @@ import typeof DeprecatedPointPropType from './Libraries/DeprecatedPropTypes/Depr
 import typeof DeprecatedViewPropTypes from './Libraries/DeprecatedPropTypes/DeprecatedViewPropTypes';
 
 import type {HostComponent as _HostComponentInternal} from './Libraries/Renderer/shims/ReactNativeTypes';
+import {DatePicker} from './Libraries/Components/DatePicker/DatePicker';
```

Comment:
I don't think this does what you were hoping for. 

1. This is coming from the compiled Typescript. Flow doesn't understand Typescript types, so Flow will see "DatePicker" as "any". We're wanting to write internals in Flow going forward to prevent these cases where we lose type safety. 

2. Doing an import here means we always load  DatePicker when loading anything else from RN. The current model is to load everything lazily, which is why the require happens on property access and we see "import type" instead of import in the above places.